query/src/main/java/org/infinispan/query/impl/massindex/IndexWorker.java

```diff
@@ -73,16 +63,14 @@ public Void apply(EmbeddedCacheManager embeddedCacheManager) {
       TimeService timeService = ComponentRegistryUtils.getTimeService(cache);
 
       IndexUpdater indexUpdater = new IndexUpdater(searchIntegrator, keyTransformationHandler, timeService);
-      ClusteringDependentLogic clusteringDependentLogic = SecurityActions.getClusteringDependentLogic(cache);
       KeyPartitioner keyPartitioner = ComponentRegistryUtils.getKeyPartitioner(cache);
 
       if (keys == null || keys.size() == 0) {
          preIndex(indexUpdater);
          if (!skipIndex) {
-            KeyValueFilter filter = getFilter(clusteringDependentLogic, reindexCache.getKeyDataConversion());
             try (Stream<CacheEntry<Object, Object>> stream = reindexCache.getAdvancedCache().withFlags(Flag.CACHE_MODE_LOCAL)
                   .cacheEntrySet().stream()) {
-               Iterator<CacheEntry<Object, Object>> iterator = stream.filter(CacheFilters.predicate(filter)).iterator();
+               Iterator<CacheEntry<Object, Object>> iterator = stream.filter(CacheFilters.predicate(AcceptAllKeyValueFilter.getInstance())).iterator();
```

Comment:
This filtering does not have any purpose now.

Reply:
right, clearly I did not finish all the possible cleanups, since ES was the last thing holding all the support for shared indexes

Reply:
fixed